packages/publish/src/lib/pack-directory.ts

```diff
@@ -69,6 +73,47 @@ export async function packDirectory(_pkg: Package, dir: string, options: PackCon
   );
 }
 
+export const PUBLISH_CONFIG_FIELDS = [
```

Comment:
it looks like there's also a "typesVersion" that pnpm can also override as defined in their docs
https://pnpm.io/package_json#publishconfig